### PR TITLE
fix template

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/478
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1245

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/page/TemplateView.coffee [hunk 1]
```diff
@@ -8,7 +8,7 @@ TemplateView = OrchestraView.extend(
     ])
     @options.configuration = @options.template
     @options.entityType = 'template'
-    @options.editable = false
+    @options.editable = true
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/coffee/page/TemplateView.coffee]: shouldn't this parameter come from the templateFacade

**Comment by itkg-alavieille** [comment]
@nicolasThal

File: ApiBundle/Transformer/TemplateTransformer.php [hunk 2]
```diff
@@ -38,6 +38,7 @@ public function transform($template)
         $facade->language = $template->getLanguage();
         $facade->deleted = $template->isDeleted();
         $facade->boDirection = $template->getBoDirection();
+        $facade->editable = true;
```
Comment (nicolasThal) [inline ApiBundle/Transformer/TemplateTransformer.php]: Isn't there a role ROLE_ACCESS_EDIT_TEMPLATE ?

**Comment by itkg-alavieille** [comment]
@nicolasThal

File: ApiBundle/Transformer/TemplateTransformer.php [hunk 3]
```diff
@@ -38,6 +38,7 @@ public function transform($template)
         $facade->language = $template->getLanguage();
         $facade->deleted = $template->isDeleted();
         $facade->boDirection = $template->getBoDirection();
+        $facade->editable = $this->authorizationChecker->isGranted(TreeTemplatePanelStrategy::ROLE_ACCESS_TREE_TEMPLATE, $template);
```
Comment (nicolasThal) [inline ApiBundle/Transformer/TemplateTransformer.php]: ROLE_ACCESS_UPDATE_TEMPLATE plutot ?

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**